### ISSUE TITLE
Feature Request - camera stream

### ISSUE THREAD

**Issue by avhm** [amosyuen/ha-mikucare#27]
I am almost certain you’ve probably looked into this already, but I’d love to be able to access the camera stream in HA.

**Comment by amosyuen** [comment]
Yeah, I looked into it a bit but seems like it requires some RTSP connection. There might be a library that can help, but seems like it isn't very straight forward.

**Comment by CajuCLC** [comment]
Miku is very closed down. And their devs are no help.

**Comment by avhm** [comment]
With the news that Miku is filing for bankruptcy, the only hope I guess we have is that they may open up the hardware or give us something to guide an open source implementation of the control system.

From the docs they filed it seems that most of the processing work was being done on AWS, and at high cost. Strange as I’m certain this was marketed as ‘on device’.

If you have any contacts there @CajuCLC now might be a good time to ask about opening the platform up.
